packages/arb-provider-ethers/src/lib/client.ts

```diff
@@ -400,13 +405,57 @@ interface MessageResult {
   evmVal: EVMResult
 }
 
+interface OutputMessage {
+  outputMsg: ArbValue.Value
+  vmId: string
```

Comment:
Is there any reason the `vmId` field is included? If not it'd be cleaner to not have it